Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CAaux.h

```diff
@@ -2,9 +2,12 @@
 #define ALIITSUCACELL_H
 
 #include <vector>
+#include <array>
 #include <cmath>
 #include "DetectorsBase/Track.h"
 #include "DetectorsBase/Constants.h"
+using std::array;
```

Comment:
This is against our coding conventions:

https://htmlpreview.github.io/?https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html#Using_declarations_and_directives